ambry-tools/src/main/java/com.github.ambry/tools/perf/rest/PerfRouter.java

```diff
@@ -156,21 +156,22 @@ public void onCompletion(Long result, Exception exception) {
   /**
    * Does nothing. Simply indicates success immediately.
    * @param blobId (ignored).
+   * @param serviceId (ignored).
    * @return a {@link FutureResult} that will eventually contain the result of the operation.
    */
   @Override
-  public Future<Void> deleteBlob(String blobId) {
-    return deleteBlob(blobId, null);
+  public Future<Void> deleteBlob(String blobId, String serviceId) {
+    return deleteBlob(blobId, serviceId, null);
   }
 
   /**
    * Does nothing. Simply indicates success immediately.
    * @param blobId (ignored).
-   * @param callback the {@link Callback} to invoke on operation completion.
-   * @return a {@link FutureResult} that will eventually contain the result of the operation.
+   * @param serviceId (ignored).
+   *@param callback the {@link Callback} to invoke on operation completion.  @return a {@link FutureResult} that will eventually contain the result of the operation.
```

Comment:
fixed, for some reason my formatter does not fix comments that go over the line length limit.